Bundle/WidgetMapBundle/Resources/config/services.yml

```diff
@@ -5,9 +5,17 @@ services:
     victoire_widget_map.widget_data_warmer:
         class: Victoire\Bundle\WidgetMapBundle\Warmer\WidgetDataWarmer
         arguments:
-            - "@annotation_reader"
-            - "@victoire_view_reference.repository"
-            - ["\Victoire\Bundle\MediaBundle\Entity\Media"]
+            - @annotation_reader
+            - @victoire_view_reference.repository
+            - @victoire_widget.widget_helper
+            - WidgetSliderItem: WidgetSlider
```

Comment:
I suggested to use Doctrine ClassMetadata to exclude automatically ManyToOne relations that are not the owning side. This hardcoded configuration will be obsolete. @gregumo you think it doable ?